ipi/utils/instools.py

```diff
@@ -64,7 +104,7 @@ def invmul_banded(A, B, posdef=False):
 
     try:
         from scipy import linalg
-        info("Import of scipy successful", verbosity.medium)
+        info("Import of scipy successful", verbosity.high)
     except ImportError:
```

Comment:
Can we discuss this scipy dependency here? Did you think about fallback options? Do you really need it?